crypto/bn/asm/x86_64-mont.pl

```diff
@@ -350,8 +368,15 @@
 .type	bn_mul4x_mont,\@function,6
 .align	16
 bn_mul4x_mont:
+.cfi_startproc
+___
+$code.=<<___ if ($addx);
+	mov	OPENSSL_ia32cap_P+8(%rip),%r11d
```

Comment:
This looks like it might be an unrelated fix. But, also, is `bn_mul4x_mont` ever called? I can't find a callsite.